packages/node_modules/@ciscospark/internal-plugin-encryption/README.md

```diff
@@ -1,3 +1,81 @@
 # @ciscospark/internal-plugin-encryption
 
-See https://ciscospark.github.io/spark-js-sdk/
+> (internal-only) Encryption for Cisco Spark
+
+This plugin provides encryption primitives (`encryptText`, `decryptBinary`, etc) and methods for interacting with a KMS by way of the Cisco Spark Encryption Service.
+
+## Security
+
+Key-fetch round trips are expensive. This library supports optional on-disk caching; you'll need to weigh the performance benefits (mostly at app launch time) vs an increase in attack area when considering use of on-disk key caching. If you only need a few keys per user occasionally, you can probably do without caching, but you're going to use a lot of the same keys regularly, the cache is going save you _a lot_ of perceived user time.
+
+## Background?
+
+## Install
+
+```bash
+npm install @ciscospark/spark-core @ciscospark/internal-plugin-encryption @ciscospark/plugin-logger
+```
+
+## Usage
+
+```js
+import '@ciscospark/internal-plugin-encryption'
+import '@ciscospark/plugin-logger'
+import SparkCore from '@ciscospark/spark-core'
+
+const spark = new SparkCore();
+
+// set auth here
+
+spark.mercury.connect()
+  .then(() => {
+    spark.internal.encryption.kms.createUnboundKeys({count: 1})
+      .then(([key]) => {
+        return spark.internal.encryption.kms.createResource({
+          userIds: [],
+          key
+        })
+      })
+      .then((kro) => {
+
```

Comment:
Maybe some comments as to what the return values for all of these are?

Reply:
ah crap. i thought that docs thing was done when I pulled it in here. apparently I never finished it...

I think I'll be pulling that out entirely for now.